proxysql/README.md

```diff
@@ -0,0 +1,112 @@
+
+2. [Restart the Agent][6].
+
+##### Log Collection
+
+1. Collecting logs is disabled by default in the Datadog Agent, enable it in your `datadog.yaml` file:
+
+   ```yaml
+   logs_enabled: true
+   ```
+
+2. Add the log files you are interested in to your `proxysql.d/conf.yaml` file to start collecting your ProxySQL logs:
+
+   ```yaml
+     logs:
+         # Default logging file
+       - type: file
+         path: /var/log/proxysql.log
+         source: proxysql
+         service: "<SERVICE_NAME>"
+         # Logged queries, file needs to be in JSON
+         # https://github.com/sysown/proxysql/wiki/Query-Logging
+       - type: file
+         path: "<QUERY_LOGGING_FILE_PATH>"
+         source: proxysql
+         service: "<SERVICE_NAME>"
+         # Audit log
+         # https://github.com/sysown/proxysql/wiki/Audit-log
+       - type: file
+         path: "<AUDIT_LOG_FILE_PATH>"
+         source: proxysql
+         service: "<SERVICE_NAME>"
+   ```
+
+    Change the `path` and `service` parameter values and configure them for your environment. See the [sample proxysql.d/conf.yaml][5] for all available configuration options.
+
+3. [Restart the Agent][6].
+
+#### Containerized
+
+For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying the parameters below.
+
+#### Metric Collection
+
+| Parameter            | Value                                                      |
+|----------------------|------------------------------------------------------------|
+| `<INTEGRATION_NAME>` | `proxysql`                                                   |
+| `<INIT_CONFIG>`      | blank or `{}`                                              |
+| `<INSTANCE_CONFIG>`  | `{"host": "%%host%%", "port": "%%port%%", "username": "<USER>", "password": "<PASSWORD>"}`       |
+
+##### Log collection
+
+Collecting logs is disabled by default in the Datadog Agent. To enable it, see [Docker log collection][9].
+
+| Parameter      | Value                                     |
+|----------------|-------------------------------------------|
+| `<LOG_CONFIG>` | `{"source": "proxysql", "service": "<SERVICE_NAME>"}` |
+
+
+### Validation
+
+[Run the Agent's status subcommand][7] and look for `proxysql` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][8] for a list of metrics provided by this check.
+
+### Service Checks
+
+`proxysql.can_connect`: Returns `CRITICAL` if the Agent can't connect to ProxySQL, otherwise returns `OK`. This service check is tagged by `server` and `port`
+
+`proxysql.backend.status`: Returns `CRITICAL` if ProxySQL considers the backend host as SHUNNED or OFFLINE_HARD. Returns `WARNING` if the backend host is `OFFLINE_SOFT`. Returns `OK` otherwise. This service check is tagged by `hostgroup`, `srv_host` and `srv_port`.
+
+### Events
+
+The ProxySQL check does not include any events.
+
+## Troubleshooting
+
+Need help? Contact [Datadog support][7].
```

Comment:
Last nit I could find: this link is wrong.

Reply:
Thanks, missed it